AN-27024 (#118)
libremfg/rhize-docs: content/reference/audit-configuration.md

| Storage system for the configuration data. Value options include: `influxdb` and `pg`.                                                                                                                              |                              

## influxdb

 A time-series database that is used in conjunction with Grafana designed for handling time-stamped data, such as metrics, events, and logs, that change over time.

| Attributes          | Description                                                                                                                                                                                    |
|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `serverUrl`         | The URL of the InfluxDB server. <br />(Default: `http://localhost:8086`)                                                                                                                       |    

> | `token`             | The authentication token used to authenticate requests to the InfluxDB server. <br />(Default: `my-token`)                                                                                     |               

Reviewer: ```suggestion
| `token`             | The authentication token to authenticate requests to the InfluxDB server. <br />(Default: `my-token`)                                                                                     |               
```

## pg

 PostgreSQL is a general-purpose relational database management system that supports a wide range of features and data types.

| Attributes          | Description                                                                                                                                                                                    |
|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `host`              | The host name of the PostgreSQL database server to which the client application will connect. <br />(Default: `dbname`)                                                                        |    

Reviewer: ```suggestion
| `host`              | The host name of the PostgreSQL database server to which the client application connects. <br />(Default: `dbname`)                                                                        |    
```

> | `user`              | The username used to authenticate with the PostgreSQL database server.                                                                                             |         

Reviewer: ```suggestion
| `user`              | The username to authenticate with the PostgreSQL database server.                                                                                             |         
```
